### PR TITLE
Redirect to display page in tables

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: pages/data-providers/[data-provider-id]/deposit-dates.jsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import dayjs from 'dayjs'
+import Icon from '@oacore/design/lib/components/icon'
```
[the remark this answers is not in the document]
Comment (Joozty) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: Oh it was done automatically by my IDE. Thanks

File: pages/data-providers/[data-provider-id]/deposit-dates.jsx [hunk 2]
```diff
@@ -50,10 +54,35 @@ const tableConfig = {
...
   ],
+  expandedRow: {
+    render: ({ content: { originalId, authors, title } }) => (
+      <div>
+        <p>
+          <b>{title}</b>
+        </p>
+        <p>{authors.map(a => a.name).join(' ')}</p>
+        <a
+          href={`https://core.ac.uk/display/${originalId}`}
```
[the remark this answers is not in the document]
Comment (Joozty) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: Isn't it the same as logout? And you said a completely different statement about it.

[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: Yeah, you are probably right. We have to move all the URLs to a configuration properties somewhere.

File: pages/data-providers/[data-provider-id]/deposit-dates.jsx [hunk 3]
```diff
@@ -50,10 +54,35 @@ const tableConfig = {
...
+          // We want to pass referrer tag to our website
+          // eslint-disable-next-line react/jsx-no-target-blank
+          target="_blank"
+          rel="noopener"
+        >
+          <Alert variant="info">
+            <Alert.Header>
+              <Icon src="/design/icons.svg#download" aria-hidden />
+              Show in CORE
```
[the remark this answers is not in the document]
Comment (Joozty) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: What ICON do you suggest? This one https://github.com/oacore/design/blob/master/src/assets/icons/outline.svg?

File: pages/data-providers/[data-provider-id]/deposit-dates.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Joozty) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: Ok I will keep it in my mind. But it is out of scope of this PR.

File: pages/data-providers/[data-provider-id]/content.jsx [hunk 4]
```diff
@@ -74,20 +74,26 @@ const tableConfig = {
...
         <div>
           <p>
             <b>{title}</b>
           </p>
           <p>{author.map(a => a.name).join(' ')}</p>
-          {fullText != null ? (
-            // We want to pass referrer tag to our website
-            // eslint-disable-next-line react/jsx-no-target-blank
-            <a href={fullText.url} target="_blank" rel="noopener">
+          {fullText || displayPage ? (
```
[the remark this answers is not in the document]
Comment (Joozty) [reply pages/data-providers/[data-provider-id]/content.jsx]: Fair enought. Well, I didn't want to repeat myself and I didn't want to extract it to new component. If it really causes confusion to you I can change it but let's be honest here. It's just 20 lines of code. I guess everybody can understand that.

File: pages/data-providers/[data-provider-id]/deposit-dates.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: If you are tied to design, do not use icon at all or use just arrow.

Also, you can add any icon from material design icons to `/public/images/icons` and use locally generated sprite file. Check it on [Iconify](https://iconify.design/icon-sets/mdi/) (document, web or something).

[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: [Open in new](https://iconify.design/icon-sets/mdi/open-in-new.html) may work.

File: pages/data-providers/[data-provider-id]/content.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply pages/data-providers/[data-provider-id]/content.jsx]: It does not confuse me and I am sure both of us can easily read it. However, you never know whom you will work with.

Also, it's not correct logically. You repeat the condition.

File: pages/data-providers/[data-provider-id]/deposit-dates.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Joozty) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: I used `document`. Didn't notice your comment. I gotta go to school. I will change it to your suggestion later.

[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: Document is fine too. Thank you

[the remark this answers is not in the document]
Comment (Joozty) [reply pages/data-providers/[data-provider-id]/deposit-dates.jsx]: Ok then...

File: components/expanded-row-alert/index.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+import { Alert, Icon } from 'design'
+
+const ExpandedRowAlert = React.memo(({ fullText, displayPage }) => {
```
Comment (viktor-yakubiv) [inline components/expanded-row-alert/index.jsx]: Sorry, I was not clear enough.

```jsx
<ExpandedRow>
  ...
  {fullText ? (
    <DocumentLink href={fullText.url}>Open full text</DocumentLink>
  ) : (
    <DocumentLink href={displayPage.url}>Open metadata</DocumentLink>
  )}
  ...
</ExpandedRow>
```

To simplify your life you can keep document icon for both.

**Pull request merged**